components/Table/SortTableHead.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Svg } from 'react-optimized-image';
+
+import SortDownIcon from 'assets/svg/app/caret-down.svg';
+import SortUpIcon from 'assets/svg/app/caret-up.svg';
+
+type SortTableHeadProps = {
+	sortable: boolean;
+	isSorted: boolean;
+	isSortedDesc: boolean;
+};
+
+export const SortTableHead: FC<SortTableHeadProps> = ({ sortable, isSorted, isSortedDesc }) => {
+	if (!sortable) return null;
+
+	if (!isSorted) {
+		return (
+			<SortIconContainer>
+				<StyledSortIcon src={SortUpIcon} viewBox={`0 0 ${SortUpIcon.width} ${SortUpIcon.height}`} />
+				<StyledSortIcon
+					src={SortDownIcon}
+					viewBox={`0 0 ${SortDownIcon.width} ${SortDownIcon.height}`}
+				/>
+			</SortIconContainer>
+		);
+	}
+
+	if (isSorted && isSortedDesc) {
```

Comment:
I'm pretty sure the `isSorted` check is needed.. not sure why it's complaining.

Reply:
Since you do an early return on line 18 when isSorted is false we know that isSorted is gonna be true here. 
```
if (!isSorted) {
		return (...)
```
So you can remove it!

Reply:
true! Done.